lib/grape/helpers/active_model_serializers.rb

```diff
@@ -8,6 +11,12 @@ module ActiveModelSerializers
       #
       # Example: To include pagination meta data: render(posts, meta: { page: posts.page, total_pages: posts.total_pages })
       def render(resource, active_model_serializer_options = {})
+        active_model_serializer_options.fetch(:serialization_context) do
+          active_model_serializer_options[:serialization_context] = ::ActiveModelSerializers::SerializationContext.new(
+            original_url: request.url[/\A[^?]+/],
```

Comment:
oops :)  

Also 👍 to making a PR to my branch and moving this code to the formatter (and adding yourself in the changelog!) https://github.com/rails-api/active_model_serializers/pull/1754#issuecomment-222990205